lib/binance_api/brokerage.rb

```diff
@@ -50,18 +56,24 @@ def validate_response(response)
     end
 
     def build_result(response)
-      json = JSON.parse(response.body, symbolize_names: true)
+      json = JSON.parse(response.body&.empty? ? '{}' : response.body, symbolize_names: true)
```

Comment:
`response.body&.empty? ? '{}' : response.body` 
not better use simper: `response.body.presence || '{}'`?